RecoVertex/VertexPrimitives/interface/VertexState.h

```diff
@@ -11,42 +11,62 @@
  */
 
 class VertexState final : private  BasicVertexState::Proxy {
-
+  
   using Base =  BasicVertexState::Proxy;
   using BSVS =  BasicSingleVertexState;
-public:
-  VertexState(){}
+ public:
+  VertexState() :
+  Base ( new BSVS () ) {}
   VertexState(VertexState const&) = default; 
   VertexState(VertexState &&) = default;
   VertexState & operator=(const VertexState&) = default;
   VertexState & operator=(VertexState&&) = default;
-
+  
   // template<typename... Args>
   //  VertexState(Args && ...args) :
   //  Base ( new BSVS ( std::forward<Args>(args)...)){}
-
+  
   explicit VertexState(BasicVertexState* p) : 
-    Base(p) {}
+  Base(p) {}    
   
   explicit VertexState(const reco::BeamSpot& beamSpot) :
-    Base ( new BSVS ( GlobalPoint(Basic3DVector<float> (beamSpot.position())), 
-		      GlobalError(beamSpot.rotatedCovariance3D()), 1.0)) {}
+  Base ( new BSVS ( GlobalPoint(Basic3DVector<float> (beamSpot.position())), 
+		    GlobalError(beamSpot.rotatedCovariance3D()), 1.0)) {}
+  
   
-
   VertexState(const GlobalPoint & pos, 
-	      const GlobalError & posErr, const double & weightInMix= 1.0) :
-    Base ( new BSVS (pos, posErr, weightInMix)) {}
+	      const GlobalError & posErr, 
+	      const double & weightInMix= 1.0) :
+  Base ( new BSVS (pos, posErr, weightInMix)) {}
   
   VertexState(const GlobalPoint & pos, 
-	      const GlobalWeight & posWeight, const double & weightInMix= 1.0) :
-    Base ( new BSVS (pos, posWeight, weightInMix)) {}
+	      const GlobalWeight & posWeight, 
+	      const double & weightInMix= 1.0) :
+  Base ( new BSVS (pos, posWeight, weightInMix)) {}
   
   VertexState(const AlgebraicVector3 & weightTimesPosition,
-	      const GlobalWeight & posWeight, const double & weightInMix= 1.0) :
-    Base ( new BSVS (weightTimesPosition, posWeight, weightInMix)) {}
+	      const GlobalWeight & posWeight, 
+	      const double & weightInMix= 1.0) :
+  Base ( new BSVS (weightTimesPosition, posWeight, weightInMix)) {}
```

Comment:
many line changes in this file appear to be unnecessary.
Consider to revert to preserve the original style.
The "Base ..." is a part of inline VertexState implementation. It should be indented.